code/game/objects/structures/statues.dm

```diff
@@ -177,8 +177,8 @@
 	else
 		return ..()
 
-/obj/structure/statue/plasma/proc/PlasmaBurn()
-	atmos_spawn_air("plasma=400;TEMP=1000")
+/obj/structure/statue/plasma/proc/PlasmaBurn(exposed_temperature)
+	atmos_spawn_air("plasma=oreAmount*10;TEMP=[exposed_temperature]")
```

Comment:
eeeeh I think `oreAmount*10` is supposed to be enclosed with brackets

Reply:
Also oreAmount is fixed so why even use it

Reply:
always prefer a variable if it exists